content/en/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch.md

```diff
@@ -382,6 +382,81 @@ You can also see the `retainKeys` strategy in the [OpenApi spec](https://raw.git
 And you can see the `retainKeys` strategy in the
 [Kubernetes API documentation](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#deploymentspec-v1-apps).
 
+## Use subresource flag with kubectl patch command to update the deployment
+
+In kubernetes v1.24, kubectl commands like get, patch, edit and replace will now contain a new flag `--subresource=[subresource-name]` which will allow fetching and updating `status` and `scale` subresources for all resources (built-in and CRs) that support these subresources.
```

Comment:
I'm not happy with this text. It sounds like it was written before Kubernetes v1.24 was released.
Instead, please describe how the latest Kubernetes release works, right now. Explain that this is generic across any API with a `status` or `scale` subresource, and that the Deployment is an example.